lib/smart_proxy_remote_execution_ssh/plugin.rb

```diff
@@ -12,6 +12,7 @@ class Plugin < Proxy::Plugin
 
     settings_file "remote_execution_ssh.yml"
     default_settings :ssh_identity_key_file   => '~/.ssh/id_rsa_foreman_proxy',
+                     :ssh_ca_known_hosts_file => '~/.ssh/ca_known_hosts',
```

Comment:
Usually we also add all the newly introduced settings which don't have a default value as comments

Reply:
Added.